fendermint/vm/interpreter/src/fvm/check.rs

```diff
@@ -109,8 +117,25 @@ where
                     // Instead of modifying just the partial state, we will execute the call in earnest.
                     // This is required for fully supporting the Ethereum API "pending" queries, if that's needed.
 
+                    let start = Instant::now();
                     // This will stack the effect for subsequent transactions added to the mempool.
                     let (apply_ret, _) = state.execute_explicit(msg.clone())?;
+                    let latency = start.elapsed().as_secs_f64();
```

Comment:
Why not create a wrapper for `state`? Like `ElapsedExecution(state).execute_explicit`, then put the logic into `ElapsedExecution`, then we can even dynamically disable the elapsed execution for someone who does not need this.